src/state.rs

```diff
@@ -176,13 +177,8 @@ impl<ID: TreeId, TM: TreeMeta, A: Actor> State<ID, TM, A> {
                     // This case should never happen in normal operation
                     // because it is required that all timestamps are unique.
                     // The crdt paper does not even check for this case.
-                    //
-                    // We can throw an exception to catch it during dev/test.
-                    // #[cfg(debug_assertions)]
-                    // panic!("applying op with timestamp equal to previous op.  Every op should have a unique timestamp.");
 
-                    // Production code should just treat it as a non-op.
-                    // #[cfg(not(debug_assertions))]
+                    info!("Non unique timestamp!");
```

Comment:
Seems like this should be a warn at least.   

Also, I think it should state something like "tried to apply op with timestamp equal to previous op" because in fact this check is only finding a very specific unique constraint violation.